node-launchpad/src/components.rs

```diff
@@ -38,7 +38,7 @@ pub trait Component {
     /// # Returns
     ///
     /// * `Result<()>` - An Ok result or an error.
-    #[expect(unused_variables)]
+    #[allow(unused_variables)]
```

Comment:
can we leave these as `expect` please, otherwise we can drop the lint

Reply:
I'm not too familiar with rust, but I couldn't `cargo build` with those remaining as `expect`. Unless there's some cargo config settings that need to be set?

Reply:
Hey, I think it might be because it was a new feature that was stablizied in the latest rust version. Could you run `rustup update` && `rustup default stable` and try compiling it?

Reply:
It appears that it does allow me to build but results in a lot more orange and red in my IDE.. Also is there a way to force a rustc version so that it warns others before attempting to build? -- Looks like maybe

```
[package]
rust-version = "1.81.0"
```